yoyodyne/data/indexes.py

```diff
@@ -160,25 +160,3 @@ def target_vocab_size(self) -> int:
             return len(special.SPECIAL) + len(self.target_vocabulary)
         else:
             return 0
-
-    @property
-    def features_vocab_size(self) -> int:
-        return len(self.features_vocabulary) if self.features_vocabulary else 0
-
-    # These are also recorded in the `special` module.
```

Comment:
How do we enforce that these  always occur in the same index position for these symbols? I assume in special the index variables are set manually?

Reply:
This hasn't changed: they are defined in `special` in a certain order and then we insert them at the beginning of the index.